cfgov/cfgov/settings/test.py

```diff
@@ -24,3 +24,15 @@
         'WIDGET': 'wagtail.tests.testapp.rich_text.CustomRichTextArea',
     },
 }
+
```

Comment:
Since `settings.test` imports all from `settings.local` (including the default DummyCache for all of the caches), what's the reason for redefining the caches like this?

Reply:
Oh, oops -- this was my mistake.  I had assumed it imports from base and didn't look at the first line in this file clearly! Will update.